tools/rebasehelpers/commitchecker/commitchecker.go

```diff
@@ -30,7 +30,10 @@ func main() {
 	// TODO: ...along with subtree merges.
 	nonbumpCommits := []util.Commit{}
 	for _, commit := range commits {
-		lastDescriptionLine := commit.Description[len(commit.Description)-1]
+		var lastDescriptionLine string
```

Comment:
How this commit relates to the generated clientsets? I think I'd prefer seeing this as a separate PR.

Reply:
funny I don't remember touching this ;)

Reply:
interesting, seems like github screwed up ;)